docs/installation/airgap.md

```diff
@@ -37,10 +37,27 @@ This method requires you to manually deploy the necessary images to each node, a
 2. Download the images archive to the agent's images directory, for example:
   ```bash
   sudo mkdir -p /var/lib/rancher/k3s/agent/images/
-  sudo curl -L -o /var/lib/rancher/k3s/agent/images/k3s-airgap-images-amd64.tar.zst "https://github.com/k3s-io/k3s/releases/download/v1.29.1-rc2%2Bk3s1/k3s-airgap-images-amd64.tar.zst"
+  sudo curl -L -o /var/lib/rancher/k3s/agent/images/k3s-airgap-images-amd64.tar.zst "https://github.com/k3s-io/k3s/releases/download/v1.33.1%2Bk3s1/k3s-airgap-images-amd64.tar.zst"
   ```
 3. Proceed to the [Install K3s](#install-k3s) section below.
 
+#### Only import if Airgap Image Tarball is new
+
+All images are imported every time k3s starts. This is done to ensure that all the images are available every time, even if some images are removed or pruned since last startup. However, this delays startup as the kubelet is not started until after all images have been imported.
+
+To alleviate this delay, starting with 1.33.1, 1.32.5, 1.31.9 and 1.30.13, there is an option to only import tarballs that have changed since they were last imported, even across restarts.
```

Comment:
Use an `:::info Version Gate` block at the top of this section, instead of listing the versions here in the paragraph.

example: https://github.com/k3s-io/docs/blob/8d88f3037ad9c4827f0681db59bc2c07343f43f6/docs/cli/etcd-snapshot.md?plain=1#L90-L92

Reply:
Good idea